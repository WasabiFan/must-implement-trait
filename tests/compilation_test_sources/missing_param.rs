use must_implement_trait::must_implement_trait;

fn main() {}

#[must_implement_trait]
struct MyStruct {
    value: i32
}