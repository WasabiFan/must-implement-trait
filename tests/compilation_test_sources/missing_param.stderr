error: expected at least one trait name as an argument; none provided.
 --> $DIR/missing_param.rs:5:1
  |
5 | #[must_implement_trait]
  | ^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
